main.cpp

```diff
@@ -260,12 +257,12 @@ void Motorway::calculateAndPrintChanceOfAccident()
             std::cout << "Chance of an accident on this stretch of motorway is  " << chanceOfAccident << " measured in accidents per hour" << std::endl;
 }
 int main() {
-    Car ford("ford", 70, 3);
-    Motorbike harley("harley", 80, 2);
-    Motorbike ducati("ducati", 110, 2);
-    Lorry bedford("bedford", 50, 3);
-    Car ferrari("ferrari", 90, 1);
-    Car vwGolf("vwGolf", 65, 5);
+    Car ford("ford", 70);
+    Motorbike harley("harley", 80);
+    Motorbike ducati("ducati", 110);
+    Lorry bedford("bedford", 50);
+    Car ferrari("ferrari", 90);
+    Car vwGolf("vwGolf", 65);
```

Comment:
Revert this to your previous main which had the number of passengers being passed as a parameter.
- [ ] revise